### PR TITLE
mamacro should be in devDependencies

### LINKED ISSUE #453: mamacro should be in devDependecies
See https://github.com/xtuc/webassemblyjs/blob/master/packages/ast/package.json#L18 and https://github.com/xtuc/mamacro/issues/2#issuecomment-421902240

### PR DESCRIPTION
**Pull request by kuba-kubula**
Fixes #453

Related is https://github.com/xtuc/mamacro/issues/2

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ install:
   # Get the latest stable version of Node.js
   - ps: Install-Product node $env:NODEJS_VERSION
   - choco install make
-  - yarn global add lerna
+  - yarn global add lerna@2
```
Comment (xtuc) [inline appveyor.yml]: Could you please revert this change?

**Comment by xtuc** [comment]
Otherwise this looks good to me, thanks.

File: appveyor.yml (hunk 1 above)
Comment (kuba-kubula) [reply appveyor.yml]: Sure, will rebase now, since there is newer lerna

**Comment by kuba-kubula** [comment]
@xtuc rebased and pushed, please feel free to merge 👋

**Comment by kuba-kubula** [comment]
@xtuc Thank you! Looking forward for a publish/release 💯
